stories/textbadge.stories.js

```diff
@@ -0,0 +1,33 @@
+//@flow
+import React from 'react';
+import TextBadge from '../src/lib/components/textbadge/TextBadge.component';
+import { Wrapper, Title } from './common';
+
+export default {
+  title: 'Components/TextBadge',
+  component: TextBadge,
+};
+
+export const Default = () => {
+  return (
+    <Wrapper>
+      <Title>Text Badges</Title>
+      <TextBadge
+        text='1'
+        variant='healthy'
```

Comment:
variant='statusHealthy'. And please update the following variant to the ones I mentioned just before. thanks

Reply:
done